resources/schema/MetaVariables.md

```diff
@@ -82,6 +82,10 @@ ItemGroupDef.ItemDef.Description.TranslatedText
 
 ItemGroupDef.ItemDef.Length
 
+## define_variable_mandatory
+
+[ItemGroupDef/ValueListDef].ItemRef.Mandatory
```

Comment:
This should be `ItemGroupDef.ItemRef.Mandatory`

Reply:
updated